src/app/tabs/newAdmin/resolver/containers/SetResolverContainer.js

```diff
@@ -0,0 +1,61 @@
+import { connect } from 'react-redux';
+import { multiChainResolver, publicResolver } from '../../../../adapters/configAdapter';
+
+import AddressInputContainer from '../../../../components/AddressInputComponent';
+import { setDomainResolver } from '../operations';
+import { closeMessage } from '../actions';
+
+const getSuggestions = (resolver) => {
+  const resolvers = [
+    {
+      name: 'Public Resolver',
```

Comment:
Let's remove this from suggestions.

Reply:
Okay, it has been removed (and rebased).

I am going to keep the functionality in the AddressInputComponent as it will be good for the subdomain view, something like "set me as owner", or something.

Reply:
Sorry, I meant just 'Public Resolver' option, my comment was not clear enough. I'm adding the Multichain option back.